packages/shared/lib/core/wallet/utils/formatBestMatchTokenAmount.ts

```diff
@@ -0,0 +1,12 @@
+import { formatNumber } from '@lib/currency'
+import { ITokenMetadata } from '../interfaces'
+import { formatDefaultTokenAmount } from './formatDefaultTokenAmount'
+
+export function formatBestMatchTokenAmount(amount: number, tokenMetadata: ITokenMetadata): string {
+    if (tokenMetadata.useMetricPrefix) {
+        // TODO: format numbers with the metric prefix
```

Comment:
Should we create a task for this rather than a TODO comment?

Reply:
Yeah I have a list of todos to create as tasks 👍